src/app/login/login.controller.js

```diff
@@ -0,0 +1,33 @@
+export class LoginController {
+  constructor ($scope,$firebaseObject,$firebaseAuth) {
+    'ngInject';
+    this.test="Good Morning";
+    this.ref = new Firebase("https://vyapi.firebaseio.com");
+
+    this.firebaseAuthLogin = function(){
```

Comment:
use [arrow function](https://babeljs.io/docs/learn-es2015/#arrows-and-lexical-this) here